app/views/govuk_publishing_components/components/docs/big_number.yml

```diff
@@ -0,0 +1,75 @@
+name: Big number
+description: A big number, with a label if needed, for visualising quantitative values such as the number of government departments or historical prime ministers.
+body: |
+  This component needs at least one item in the `items` array passed to it, otherwise it won't render. If no `number` attribute is passed in the item, it will default to "0".
+accessibility_criteria: |
+  - uses clear language and is positioned in a logical context on the page it's being used on
+  - communicates number value and label (if present) in a single dictation when reading with screen reader software
+shared_accessibility_criteria:
+- link
+examples:
+  default:
+    data:
+      items:
+        - number: 119
+  multiple_number_items:
+    description: |
+      If more than one number is present in the `items` array, it will be rendered as an unordered list as opposed to just the number element alone.
+    data:
+      items:
+        - number: 119
+        - number: 553
+  with_labels:
+    description: |
+      Labels for numbers, if present, are styled in such a way that they are inline but appear to have a block relationship. This is to adhere to established design whilst bypassing a minor issue for NVDA users where the software will identify block level items and dictate them separately. This this case it is less jarring for screen reader users to hear the number and the label as a single dictation.
+    data:
+      items:
+        - number: 119
+          label: Open consultations
+        - number: 553
+          label: Closed consultations in the past 12 months
+  with_plus_suffix:
+    description: |
+      In some cases, we want to communicate that a given number is the "minimum" for that value. We can do this by suffixing a plus symbol onto the desired number.
+    data:
+      items:
+        - number: 23
+          label: Ministerial departments
+        - number: 400
+          show_plus: true
+          label: Other agencies and public bodies
+  with_link:
+    description: |
+      If a `href` attribute is present on a given number, we wrap that item in an `a` tag. Only the label is underlined and underline hover styles are only applied to that label.
+    data:
+      items:
+        - number: 23
+          label: Ministerial departments
+          href: "/government/organisations#ministerial_departments"
+        - number: 400
+          show_plus: true
+          label: Other agencies and public bodies
+          href: "/government/organisations#agencies_and_other_public_bodies"
+  with_link_but_no_label:
+    description: |
+      In the event that a `href` is provided but a `label` isn't, a decorator class is applied to the number itself that mirrors the `govuk-link--no-underline` class.
```

Comment:
I think this could be written more clearly...

```
Numbers that are links are underlined unless a label is provided, in which case the label is given the underline.
```